src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -1984,13 +1984,18 @@ def test_vm_accelerated_networking(self, resource_group):
         self.kwargs.update({
             'vm': 'vm1'
         })
+        # TODO: this relies on validation in _validate_vm_vmss_accelerated_networking to implicitly enable accelerated networking.
```

Comment:
This doesn't look like a `todo`, and I would suggest the comment updated to be `cli's defaulting logic would turn on the accelerated networking based on the vm size and os image`.

Reply:
yes, that is pending #6440

Reply:
Why does this make the command flaky?

Reply:
Because information like [this](https://github.com/Azure/azure-cli/pull/8438/files#diff-c4a054c29f906bf64c5d4e349ce38f49R670) is hardcoded and need to be updated whenever there are service changes.